omniNotes/src/androidTest/java/it/feio/android/omninotes/Test/AppointmentDetailsTest.java

```diff
@@ -0,0 +1,85 @@
+        assertTrue("Add button is not displayed.", Home.isAddButtonVisible());
+        assertTrue("Add button is not clickable.", Home.isAddButtonClickable());
+
+        // Step 2. Click "Add" button
+        Home.clickAddButton();
+
+        // Expected Result: Test "Checklist" is between 2 other texts.
+        assertTrue("Text \"Checklist\" is not above Text note text.", TextNote.isChecklistTextAboveTextNoteText());
+        assertTrue("Text \"Checklist\" is not below Photo Text.", TextNote.isChecklistTextBelowPhotoText());
+
+        // Expected Result: The suggestive picture of "Photo" text is above "Checklist" suggestive picture.
+        assertTrue("The suggestive picture of \"Photo\" text is NOT above \"Checklist\" suggestive picture.", Home.isSuggestivePictureOfPhotoAboveChecklistSuggestivePicture());
+
+        // Step 3. Click "Text note" button.
+        TextNote.clickTextNoteButton();
+
+        // Expected Result: "Content" and "Add reminder" are descendent of a LinearLayout and their parents are siblings.
+        assertTrue("\"Content\" is NOT a descendant of a LinearLayout.", Home.isContentBoxDescendantOfALinearLayout());
+        assertTrue("\"Add reminder\" is NOT a descendant of a LinearLayout.", Home.isAddReminderDescendantOfALinearLayout());
+        assertTrue("The \"Content parent and the \"Add reminder\" parent are NOT siblings.", Home.isContentParentSiblingWithAddReminder());
+
+        // Step 4. Click "Title" container and Type "Doctor appointment".
+        TextNote.clickTitleBox();
+        TextNote.typeAppointmentName();
+
+        // Step 5. Click "Content" container and type " Tuesday, 17 Jul 2018, 18:30.".
+        TextNote.clickContentBox();
+        TextNote.typeAppointmentContent();
+
+        // Step 6. Click on "Add reminder" and Add a reminder for the date from "Content" container.
+        TimePiker.clickOnAddReminderButton();
+        TimePiker.clickOnAnyDayFromTimePiker();
+
+        // Step 7. Click "Done" button.
+        TimePiker.clickOnTimePikerDoneButton();
+
+        // Expected Result: The "Time picker" is displayed.
+        assertTrue("The \"Time picker\" is NOT displayed.", TimePiker.isTimePikerVisible());
+
+        // Step 8. Click "Done" button.
+        TimePiker.clickOnClockDoneButton();
+
+        // Step 9. Swipe the "repeat weekly" button.
+        TimePiker.swipeRightTheRepeatWeeklyButton();
+
+        // Expected Result: The number of weeks is between 2 texts.
+        assertTrue("The interval is NOT in the left of \"week\" text.", TimePiker.isIntervalInTheLeftOfWeekText());
+        assertTrue("The interval is NOT in the right of \"Every\" text.", TimePiker.isIntervalInTheRightOfEveryText());
+
+        // Step 10. Click on "Friday" button.
+        TimePiker.clickOnFridayButton();
+
+        // Step 11. Click on "Done" button.
+        TimePiker.clickOnTimePikerDoneButton();
+
+        // Step 12. Press back.
+        TextNote.clickReturnButton();
+        Thread.sleep(2000);
```

Comment:
This is not a step. You can add a comment before this.

Reply:
Done